src/auth/exceptions/auth.exception.ts

```diff
@@ -0,0 +1,6 @@
+export class AuthException extends Error {
+    constructor(message: string, options?: any) {
+        super(message, options);
+    }
+    name = 'AuthException';
```

Comment:
czemu to pole nie ma zadnych modyfikatorow?

Reply:
ddoam